generators/client/templates/angular/package.json.ejs

```diff
@@ -40,7 +40,6 @@
     "@ng-bootstrap/ng-bootstrap": "3.0.0",
     "bootstrap": "4.1.3",
     "core-js": "2.5.7",
-    "jquery": "3.3.1",
```

Comment:
This means we dont use JQuery anymore, so may be we should remove the logo from website as well @wmarques

Reply:
I'll do that tonight 🙂